bin/travis_scripts/ensure_dep.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+TOOLS_PATH=$1
+DEP_VERSION=$2
+
+
+OS_ARCH="linux-amd64"
+if [ "$TRAVIS_OS_NAME"=="osx" ]; then
+    OS_ARCH="darwin-amd64"
+fi
+
+dep --version
+if [ $? -eq 0 ]; then
+    echo "Dep found"
+else
+    # TODO: fetch frozen dep version
+    go get -u github.com/golang/dep/cmd/dep
+#    wget "https://github.com/Masterminds/glide/releases/download/$GLIDE_VERSION/glide-$GLIDE_VERSION-$OS_ARCH.tar.gz"
```

Comment:
Not needed

Reply:
What's not needed, freezing version? why?